lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -61,9 +61,10 @@
                 <span class="tip">${_("(Maximum 128 characters)")}</span>
             </label>
         </li>
-        <li>
+        <li class="field">
```

Comment:
Still needed for span css styling, inexplicably removed by upstream: https://github.com/edx/edx-platform/pull/12301/files#diff-0aafb7e04aabb73712746a2815573bebR50